migrations/versions/9d21de792967_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 9d21de792967
+Revises: 194a5a2a44ef
+Create Date: 2018-06-05 20:03:20.588368
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '9d21de792967'
+down_revision = '194a5a2a44ef'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint('ticket_event', 'ticket_holders', ['ticket_id', 'event_id'])
```

Comment:
Does this migrate well? I mean when one has a non-unique relationship in the db? Or does the migration crash then?

Reply:
@maxlorenz It crashes. I tested it locally and it crashed. How should we handle that?

Reply:
Delete all except one

Reply:
@iamareebjamal How do we decide the one to keep? The earliest one? The one with lowest `id`?

Reply:
Yes, the first one should be kept